content/800-data-platform/04-working-with-data/400-schema.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Prisma schema'
+metaTitle: 'Prisma schema in the Data Platform'
+metaDescription: 'Prisma schema in the Data Platform'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+You can view the Prisma schema of the current project environment in the **Schema** tab.
+
+Under the Schema title, you can view when your Prisma schema for the current environment was last updated indicated by the `Last pulled from GitHub 1 hour ago` line.
```

Comment:
It might be worth mentioning somewhere here that the Schema file you're viewing is pulled from the specific Github branch that you added to your current environment. A common problem our users are facing is that they are telling us that they're not seeing their schema changes deployed and that's because they push to the wrong branch so it might be worth to also add this to the list of problems in the page.

We plan to improve that by showing the github branch in the page ([ticket](https://linear.app/prisma-company/issue/CP-2327/show-gh-branch-in-the-schema-page)) but until then, the Docs might help some people out.

Reply:
(that's the only change I'm suggesting, btw, sorry for the unorthodox double PR review 😅 )

Reply:
> 

I've added this to the Schema page.

If you select another environment which uses a separate branch or repository, the Schema tab shows the schema file from that branch or repository.